### PR TITLE
Use scalars for consensus round encoding

### PR DESCRIPTION
**Pull request by rain-on**
Signed-off-by: Trent Mohay <trent.mohay@consensys.net>

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/validation/RoundChangePayloadValidatorTest.java [hunk 1]
```diff
@@ -139,18 +139,6 @@ public void roundChangeWithZeroTargetRoundFails() {
     assertThat(messageValidator.validate(signedPayload)).isFalse();
   }
 
-  @Test
-  public void roundChangeWithNegativeTargetRoundFails() {
```
Comment (jframe) [inline consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/validation/RoundChangePayloadValidatorTest.java:143]: Why remove this test?

File: consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java [hunk 2]
```diff
@@ -26,8 +26,8 @@
 import java.util.Objects;
 import java.util.StringJoiner;
 
-public class CommitPayload implements Payload {
-  private static final int TYPE = QbftV1.COMMIT;
+public class CommitPayload extends QbftPayload {
+  public static final int TYPE = QbftV1.COMMIT;
```
Comment (jframe) [inline consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java:30]: why does this need to be public? couldn't see where it is used

**Review by jframe — APPROVED** [review]
LGTM

File: consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java (hunk 2 above)
Comment (rain-on) [reply consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java]: yup - should be private, was part of a secondary piece of work I was tinkering with

**Pull request merged**